src/vm/method.hpp

```diff
@@ -1005,6 +1005,14 @@ class MethodDesc
             && GetSlot() < pMT->GetNumVirtuals();
     }
 
+    // Is this a default interface method (virtual non-abstract instance method)
+    inline BOOL IsDefaultInterfaceMethod()
+    {
+        LIMITED_METHOD_CONTRACT;
+
```

Comment:
if the #if for default interfaces isn't enabled, this should just return false.

Reply:
Fixed.